src/maintainer/infrastructure.rst

```diff
@@ -227,11 +227,8 @@ Appveyor is used to build Windows packages.  It is the only provider that can bu
 Skipping CI builds
 ------------------
 
-.. todo::
-
-  - add information regarding [ci skip] for all CIs.
-
-.. admonition:: Related links
-
-  - **abort builds with [skip ci]/etc** `(conda-forge.github.io/#629) <https://github.com/conda-forge/conda-forge.github.io/issues/629>`__
-  - **Skip CI requests** `(staged-recipes/#1148) <https://github.com/conda-forge/staged-recipes/issues/1148>`__
+Each CI provider allows for different keywords in commit titles:
```

Comment:
Can we explicitly list skip keywords here in addition to referencing to the upstream documentation?

Reply:
Yes, that would be great!